src/Spectre.libGenetic/GenerationFactory.cpp

```diff
@@ -29,7 +29,7 @@ GenerationFactory::GenerationFactory(size_t size, size_t individualSize, size_t
       m_IndividualSize(individualSize),
       m_InitialFillup(initialFillup)
 {
-    if (initialFillup > individualSize)
+    if (initialFillup > individualSize || initialFillup < 0)
```

Comment:
`initialFillup` is `size_t`. How do you set it to be negative?

Reply:
done